repose-aggregator/components/filters/keystone-v2/src/main/resources/META-INF/schema/config/keystone-v2.xsd

```diff
@@ -108,6 +108,13 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="set-rackers-groups" type="xs:string" use="optional">
```

Comment:
you've tied two things together here, which i'm not certain should be. i assume this attribute is what determines whether to let the groups call fail for rackers, but you are also using it for a value to put in. These are separate items. Only sort of though, it's an option on an option, which suggests it should be an element with an attribute. also your right the name is bad.